tests/fixtures.py

```diff
@@ -253,6 +253,8 @@ def get_raw_transaction(span_id: str | None = None) -> InsertEvent:
                     "description": "SELECT * FROM users",
                     "data": {},
                     "timestamp": calendar.timegm(end_timestamp.timetuple()),
+                    "hash": "5029609156d8133",
+                    "exclusive_time": 1.2,
```

Comment:
is this related to your change?

Reply:
Yes, I needed this data in here to test the spans query, which is the use case for this change.